src/core/janeway_global_settings.py

```diff
@@ -616,3 +637,51 @@ def __len__(self):
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',
     'PAGE_SIZE': 100,
 }
+TINYMCE_CLIPBOARD_CLEANER = {
+    # Settings required to optionally clean formatting from a paste event
+    # Install a callback to capture the original input from clipboard
+    "setup": """
+        function (editor) {
+            editor.on('paste', function (e) {
+                editor.cachedClipboardContent = e.clipboardData.getData("text/html");
+            });
+        }
+    """,
+    # Configure TinyMCE to always clean to text
+    "paste_as_text": True,
+    # Intercept paste if the input has been cleaned and confirm with user
+    "paste_preprocess": """
+        function (editor, args) {
+            if (
+                editor.cachedClipboardContent
+                && editor.cachedClipboardContent != args.content
+            ){
+                const doClean = confirm("Formatted paste detected, Click 'OK' to paste as text or 'Cancel' to keep the formatting.");
```

Comment:
Can we make this two sentences because of silly grammar rules that will annoy academics if not followed? :) 
```
Formatted paste detected. Click 'OK' to paste as text or 'Cancel' to keep the formatting.
```